ext/nnstreamer/tensor_filter/tensor_filter_openvino.cc

```diff
@@ -207,17 +275,21 @@ TensorFilterOpenvino::loadModel ()
     this->_ieCore.AddExtension(
         std::make_shared<InferenceEngine::Extensions::Cpu::CpuExtensions>(),
         "CPU");
-    this->_executableNet = this->_ieCore.LoadNetwork (this->_networkCNN,
-        *strVectorIter);
-    this->isLoaded = true;
 #endif /* __OPENVINO_CPU_EXT__ */
```

Comment:
Q: Then, even if `__OPENVINO_CPU_EXT__` is not defined, is it regarded as a successful load?

Reply:
@dongju-chae A: In that case, by the lines from 559 to 563, this method, loadModel (), never be invoked.